src/burnchains/burnchain.rs

```diff
@@ -1213,12 +1213,21 @@ impl Burnchain {
         );
 
         if let Some(target_block_height) = target_block_height_opt {
-            if target_block_height < end_block {
+            // target_block_height is used as a hint, but could also be completely off
```

Comment:
Maybe put the variable name `target_block_height` in some kind of quotes.

Reply:
```suggestion
            // `target_block_height` is used as a hint, but could also be completely off
```